src/lib/server/database/index.ts

```diff
@@ -459,6 +459,20 @@ export async function editTicketTitle(tid: Ticket['ticket_id'], title: Ticket['t
     }
 }
 
+export async function editTicketDueDate(tid: Ticket['ticket_id'], tdd: Ticket['due_date']) {
+    const time = tdd.toISOString();
+    const { count } =
+        await sql`UPDATE tickets SET due_date = ${time} WHERE ticket_id = ${tid}`.execute();
```

Comment:
I believe this should be allowed.
```suggestion
export async function editTicketDueDate(tid: Ticket['ticket_id'], due: Ticket['due_date']) {
    const { count } =
        await sql`UPDATE tickets SET due_date = ${due} WHERE ticket_id = ${tid}`.execute();
```